README.md

```diff
@@ -8,9 +8,24 @@
 
 ## Features
 
-- **Fast reading and writing** of RDF knowledge graphs in the [Jelly format](http://w3id.org/jelly)
-- **Seamless integration with [rdflib](https://rdflib.readthedocs.io/)**
-- **Stream processing support** for large datasets or streams of many RDF graphs/datasets
+- **Fast reading and writing** of RDF knowledge graphs in the [Jelly format](http://w3id.org/jelly).
+- **Standalone [generic API](generic-sink.md)** with no third-party dependencies, allowing for:
```

Comment:
that's a 404 – please link to https://w3id.org/jelly/pyjelly/something

We use w3id permanent URLs and they are redirected to github pages.